services/src/datasets/postgres.rs

```diff
@@ -554,12 +554,13 @@ where
         let conn = self.conn_pool.get().await?;
 
         conn.execute(
-            "UPDATE datasets SET name = $2, display_name = $3, description = $4 WHERE id = $1;",
+            "UPDATE datasets SET name = $2, display_name = $3, description = $4, tags = $5 WHERE id = $1;",
```

Comment:
Wouldn't it be less error-prone if we start with $1 and end with $5?

Reply:
I am not sure what is less error-prone. If we add a new field, then id becomes $6, and then $7 and so on. Right now the id is always $1 no matter how we change the query. This avoids errors as well.